setup.py

```diff
@@ -44,6 +44,6 @@
       scripts=glob('scripts/*'),
       extras_require={'test': ["nose >= 0.10.1", "pep8"]},
       install_requires=['click >= 3.3', 'future', 'requests', 'pandas >= 0.15',
-                        'h5py >= 2.3.1', 'humann2'],
+                        'h5py >= 2.3.1', 'humann2', 'biom-format'],
```

Comment:
Can you revise the actual libraries that you are importing? If you are not importing directly future, requests, pandas, h5py and biom-format they shouldn't be listed here. I think humman2 should be installing biom-format and h5py, if it is not, it probably would be better to contact them to fix their setup.py. I don't think you're using requests either, since you're using the qiita client.